From 163040415eae8c3e78a970af9f995ed46ccd1c29 Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 20 Nov 2014 15:56:39 +0000
Subject: [PATCH 0544/1000] MANTIS 34722 Mantis 34722: Report media-low over
 the "Atb2Dev.idl" interface - Extend the mediaStatusType enumeration by
 MEDIA_HIGH.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@780 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/atb2dev.idl | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SelfServiceCommon/Massai/idl/Massai/atb2dev.idl b/SelfServiceCommon/Massai/idl/Massai/atb2dev.idl
index 1afa8bc2..c6ccde42 100644
--- a/SelfServiceCommon/Massai/idl/Massai/atb2dev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/atb2dev.idl
@@ -77,6 +77,8 @@ module atb2dev
     {
       MEDIA_ABSENT,             /**< Media present in device. */
       MEDIA_PRESENT,            /**< Media not present in device. */
+      MEDIA_HIGH,               /**< Media present in device. Report this rather than MEDIA_PRESENT,
+                                     when MEDIA_LOW could possibly be reported. */
       MEDIA_LOW                 /**< Media in device is nearly gone. */
     };
 
-- 
2.41.0.windows.1

